CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 To install Beacon, see the [documentation](https://developer.helpscout.com/beacon-2/mobile/).
 
+## 3.0.0 Release (TBD) 
+
+#### Bug fixes
+
+* Xcode 15 Support.
+
+#### Deprecations
+
+* iOS/iPadOS 11 is no longer supported.
```

Comment:
What do you think about giving more context on why we're dropping iOS 11 support?

```suggestion
* **iOS/iPadOS 11 is no longer supported** because Xcode 15 requires iOS 12 or greater. For more info, [see the Xcode 15 Release Notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-15-release-notes).
```